test/Reader/Http/ResponseTest.php

```diff
@@ -0,0 +1,175 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
+ * @copyright Copyright (c) 2005-2015 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   http://framework.zend.com/license/new-bsd New BSD License
+ */
+
+namespace ZendTest\Feed\Reader\Http;
+
+use PHPUnit_Framework_TestCase as TestCase;
+use Zend\Feed\Reader\Exception\InvalidArgumentException;
+use Zend\Feed\Reader\Http\Response;
+use ZendTest\Feed\Reader\TestAsset\Psr7Stream;
+
+/**
+ * @covers \Zend\Feed\Reader\Http\Response
+ */
+class ResponseTest extends TestCase
+{
+    public function testConstructorOnlyRequiresStatusCode()
+    {
+        $response = new Response(200);
+        $this->assertEquals(200, $response->getStatusCode());
+        $this->assertEquals('', $response->getBody());
+        $this->assertAttributeSame([], 'headers', $response);
+    }
+
+    public function testConstructorCanAcceptResponseBody()
+    {
+        $response = new Response(201, 'CREATED');
+        $this->assertEquals(201, $response->getStatusCode());
+        $this->assertEquals('CREATED', $response->getBody());
+        $this->assertAttributeSame([], 'headers', $response);
+    }
+
+    public function testConstructorCanAcceptAStringCastableObjectForTheResponseBody()
+    {
+        $stream   = new Psr7Stream('BODY');
+        $response = new Response(200, $stream);
+        $this->assertEquals('BODY', $response->getBody());
+    }
+
+    public function testConstructorCanAcceptHeaders()
+    {
+        $response = new Response(204, '', [
+            'Location'         => 'http://example.org/foo',
+            'Content-Length'   => 1234,
+            'X-Content-Length' => 1234.56,
+        ]);
+        $this->assertEquals(204, $response->getStatusCode());
+        $this->assertEquals('', $response->getBody());
+        $this->assertEquals('http://example.org/foo', $response->getHeaderLine('Location'));
+        $this->assertEquals(1234, $response->getHeaderLine('Content-Length'));
+        $this->assertEquals(1234.56, $response->getHeaderLine('X-Content-Length'));
+    }
+
+    public function invalidStatusCodes()
+    {
+        foreach (range(-100, 99) as $statusCode) {
```

Comment:
While test evrry case is the best, I've found require too much time for execute the tests with coverage. 

Consider test only the boundaries https://github.com/Maks3w/zend-expressive/commit/ba945a047696695aee3cbb5f1de0497ab85b307c